src/basilisp/lang/compiler/generator.py

```diff
@@ -3527,7 +3527,7 @@ def _const_type_to_py_ast(form: IType, ctx: GeneratorContext) -> GeneratedPyAST:
 
     ctor_args = []
     ctor_arg_deps: List[ast.AST] = []
-    for field in attr.fields(tp):  # type: ignore[arg-type]
+    for field in attr.fields(tp):  # type: ignore[arg-type, misc, unused-ignore]
```

Comment:
```suggestion
    for field in attr.fields(tp):
```

I think you can just do this rather than adding `unused-ignore` code.

Reply:
Unfortunately `ignore` by itself does not suffice if there is an `unused-ignore` error reported in some python versions (e.g. 3.8) but not others (3.11), it has to be added to the list:

please see below how `type: ignore` reports an unused-ignore error for runtime.py in 3.8 but not in 3.11.

3.8:
```
py38-mypy: commands[0]> mypy --config-file=C:\src\basilisp/pyproject.toml -p basilisp
src\basilisp\lang\runtime.py:1958: error: Unused "type: ignore" comment  [unused-ignore]
src\basilisp\lang\compiler\generator.py:3530: error: Unused "type: ignore[misc]" comment  [unused-ignore]
src\basilisp\lang\compiler\analyzer.py:336: error: Unused "type: ignore" comment  [unused-ignore]
Found 3 errors in 3 files (checked 47 source files)
```

3.11
```
py311-mypy: commands[0]> mypy --config-file=C:\src\basilisp/pyproject.toml -p basilisp
src\basilisp\lang\runtime.py:1958: error: Argument 1 to "intern" of "Var" has incompatible type "Namespace | None"; expected "Namespace | Symbol"  [arg-type]
```